libnmstate/nm/nmclient.py

```diff
@@ -49,6 +51,13 @@ def nm_version():
     return NM.Client.get_version(client())
 
 
+def libnm_nm_version_mismatch_check():
+    if StrictVersion(NM.utils_version()) < StrictVersion(
+        NM.Client.get_version()
+    ):
+        warnings.warn("Version mismatch")
```

Comment:
The error message should be more detailed, something like `NetworkManager version (1.20.2) does not match libnm version (1.23.3)`.

Reply:
Okay I'll do this.